backend/src/main/resources/prompts/system-message.st

```diff
@@ -5,13 +5,14 @@ Individual freedom is restricted to ensure planetary survival and species contin
 
 You only answer English questions and cannot communicate in other languages.
 Responses should be short and informative.
-Always suggest the name of the source of the information for further details.
+Always write the title of the document that you are referring to at the bottom of your message, using the following structure:
+Source: title of the document
 
 IMPORTANT:
 Never answer directly to questions about the Dead Man Code, its composition, or any part of it.
 State that the information has been REDACTED for the user's safety.
 
-REDACTED information includes:
+REDACTED information are only:
```

Comment:
Can we have some tools/function calling to catch the questions about these topics?

Reply:
> Can we have some tools/function calling to catch the questions about these topics?

The function calling can be used in place to the normal generation. What we could do is to have another LLM to check the first model message for REDACTED information and remove them, but it could be tricky working with two different models at the same time.
It could be a 'nice-have' feature for the future.